features/edition-attachments.feature

```diff
@@ -54,3 +54,17 @@ Feature: Managing attachments on editions
     And a draft publication "Standards on Beard Grooming" exists
     When I try and upload an attachment but there are validation errors
     Then I should be able to submit the attachment without re-uploading the file
+
+  Scenario: Editing metadata on attachments
+    Given I am an writer
+    And I start drafting a new publication "Standard Beard Lengths"
+    When I start editing the attachments from the publication page
+    And I upload an html attachment with the title "Beard Length Graphs 2012" and the isbn "123456789" and the contact address "Address 1"
+
```

Comment:
There's no `Then` here, so it feels like the feature doesn't actually describe a complete scenario. I know that it *does* test something, but it doesn't really read like it does. Can we combine this test and the next one?